### ISSUE TITLE
Case Sensitive options for String Like operator

### ISSUE THREAD

**Issue by thutch** [cagrin/LikeComparison#12]
Is it possible to set the Like comparison to be Case Sensitive?  
Example

"TEST".Like("test")  
for Case sensitive, this should return a false.

**Comment by cagrin** [comment]
Yes, sure. You can use PostgreSql variant, which is case-sensitive:

```using LikeComparison.PostgreSql;```
...
```bool isLike = "TEST".Like("test"); // returns false```
